src/chrome/content/rules/hungryhowies.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="hungryhowies.com">
+        <target host="hungryhowies.com" />
+        <target host="www.hungryhowies.com" />
```

Comment:
Please also add the following targets:
franchising.hungryhowies.com
home.hungryhowies.com